editor/src/components/common/shared-strategies/convert-to-flex-strategy.ts

```diff
@@ -154,7 +154,7 @@ function guessPadding(
   const paddingLeft = firstChild.frame.x - parentRect.x
   const paddingRight =
     parentRect.x + parentRect.width - (lastChild?.frame.x + lastChild?.frame.width)
-  const horizontalPadding = Math.min(paddingLeft, paddingRight)
+  const horizontalPadding = Math.max(0, Math.min(paddingLeft, paddingRight))
```

Comment:
I think `clamp` from `math-utils` would be more readable here

Reply:
fair, fair

Reply:
retracted, the original intention was to pick the smaller of `paddingLeft` and `paddingRight`, and ensure that the resulting value is larger or equal to 0, clamp is obviously not applicable here